index.php

```diff
@@ -4,16 +4,16 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Party Czechitas</title>
+    <title>Party <?php echo "Czechitas"; ?></title>
```

Comment:
Tohle tam asi dělat nebudeme... Uvidíme, jak jim to půjde. Je title v těch lekcích k html?

Reply:
je